tests/unit/helpers/bills.test.js

```diff
@@ -1519,9 +1487,11 @@ describe('formatAndCreateBills', () => {
         sinon.assert.calledWithExactly(updateEventsStub, eventsToUpdate);
         sinon.assert.calledWithExactly(updateFundingHistoriesStub, {}, companyId);
         sinon.assert.notCalled(createBillSlips);
-        BillNumberMock.verify();
-        CreditNoteMock.verify();
-        BillMock.verify();
+        sinon.assert.calledOnceWithExactly(
+          insertManyBill,
+          [customerBillingInfo.bill, ...tppBillingInfo.tppBills]
+        );
+        sinon.assert.calledOnce(updateOneBillNumber);
```

Comment:
calledOnceWithExactly ?

Reply:
tu es sûre ? étant donné qu'il n'y a pas d'arguments ? ent tout cas ça fait planter le test

Reply:
je ne comprends pas ta reponse : pour moi il est appelé avec 2 arguments
`await BillNumber.updateOne({ prefix: number.prefix, company: company._id }, { $set: { seq: number.seq } });`

Reply:
pas côté tests en tout cas, y a juste `BillNumberMock.expects('updateOne').throws('updateOneError');` sans withExactArgs. C'est un oubli des précédents tests ?

Reply:
oui je pense ! -> cf ce qu'on s'est dit tout a l'heure : quand on repasse sur du code pour le modifier, soyons vigilant aux petites ameliorations qu'on peut faire pour le rendre encore mieux ;)

je crois qu'on a ce cas dans d'autres it egalement

Reply:
done